app/src/services/realitio.ts

```diff
@@ -157,10 +157,15 @@ class RealitioService {
   }
 
   getWinnerOutcome = async (questionId: string): Promise<number> => {
```

Comment:
Is this method still used?

Reply:
It's still being used for the "Has winning outcomes" filter... which means that that filter won't quite work with nuanced binary questions and undefined answers. I will open an issue for that.